dataland-frontend/src/components/resources/frameworkDataSearch/ShowMultipleReportsBanner.vue

```diff
@@ -7,33 +7,30 @@
     </h4>
     <div id="reportList" style="display: flex">
       <span v-for="(report, name, index) in reports[indexOfNewestReportingPeriod]" :key="index" class="link-in-list">
-        <DocumentLink
-          data-test="documentLinkTest"
-          :download-name="name"
-          :fileReference="report.fileReference"
-          font-style="font-semibold"
-        />
+        <a @click="openReportDataTableModal(report, name as string)" class="link" :data-test="`report-link-${name}`">
```

Comment:
Always show all fields about the report no matter if they are filled in or not

Reply:
done